        <model name="{obstacle_name}">
        <static>true</static>
        <pose>{x} {y} {z} 0 0 0</pose>
        <link name="link">
            <collision name="collision">
            <geometry>
                <box>
                <size>{length_x} {length_y} {obstacle_height}</size>
                </box>
            </geometry>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>{length_x} {length_y} {obstacle_height}</size>
                </box>
            </geometry>
            <material>
{color}
            </material>
            </visual>
        </link>
        </model>
